src/assets/i18n/es.json

```diff
@@ -1021,6 +1021,7 @@
     "search_last_week": "La semana pasada",
     "search_this_month": "Este mes",
     "search_last_month": "El mes pasado",
+    "select_actions": "Seleccionar Acciones",
     "no_logs": "No hay registros que coincidan con los filtros.",
```

Comment:
I think you missed Spanish and German!
Double check that the 4 files are in sync.
Thanks.